### DISCUSSION TITLE
I am in love with Admin Interface, so thanks up front.  I have a multi tenant application that I want to work with admin-interface

### DISCUSSION THREAD

**Discussion by trexmo** [fabiocaccamo/django-admin-interface#369] [Q&A]
using django_tenants I am getting errors with admin_interface theme does not exist.  How can I get a different theme for different tenants or users?

**Comment by fabiocaccamo** [comment]
@trexmo django tenants support has already been discussed in #170, #199 and #242, maybe you can find some useful informations there.

**Reply by trexmo** [reply]
Well, #170 has no solution, #199 is about signals, not tenants and #242 is about an ad hoc tenant design and not django_tenants.  perhaps someone over at django_tenant will be able to help.

**Reply by fabiocaccamo** [reply]
@trexmo I would suggest you to open an issue with a detailed explanation of the errors you're encountering, me or someone else will point you in the right direction.
